.travis.yml

```diff
@@ -1,35 +1,100 @@
 language: php
-sudo: required
-dist: trusty
 
-matrix:
+jobs:
   include:
-  - php: 7.0
-  - php: 7.1
-  - php: 7.2
-  - php: 5.5.38
-  - php: 5.6.25
-  - php: hhvm-3.24
-    group: edge
-  fast_finish: true
-
-before_script:
-  - pecl install grpc || echo 'Failed to install grpc'
-  - if [[ $TRAVIS_PHP_VERSION =~ ^7 ]]; then pecl install stackdriver_debugger-alpha || echo 'Failed to install stackdriver_debugger'; fi
-  - composer install
-  - if [[ $TRAVIS_PHP_VERSION =~ ^hhvm ]]; then composer --no-interaction --dev remove google/protobuf google/gax google/proto-client; fi
-  - dev/google-cloud credentials
+    - php: 7.2
+      install:
+        - pecl install grpc || echo 'Failed to install grpc'
```

Comment:
Maybe a style choice, but one option other than just having each build do its own install would be to have separate scripts for installing php 5, php 7, and hhvm.

Reply:
like, for instance, `./dev/sh/install-php7` to do the grpc, stackdriver_debugger and composer install steps?

Reply:
Yes. But that is only a thought, feel free to stick to this approach if you prefer.